client/src/lib/hooks/useConnection.ts

```diff
@@ -297,7 +297,12 @@ export function useConnection({
         bearerToken || (await serverAuthProvider.tokens())?.access_token;
       if (token) {
         const authHeaderName = headerName || "Authorization";
-        headers[authHeaderName] = `Bearer ${token}`;
+        headers[authHeaderName] = token;
```

Comment:
We don't know that `authHeaderName` is not "Authorization" here. If it is, this value should be `Bearer ${token}`.

Reply:
resolved